### PR TITLE
New external id dynamic on Fundings and Works

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: src/locale/properties/shared/shared.en.properties [hunk 1]
```diff
@@ -144,3 +144,6 @@ shared.worksDescription=Add your research outputs such as publications, data set
 shared.worksLearnMore=Learn more about adding works to your ORCID record
 shared.employmentDescription=Add details of your current and previous employers.
 shared.employmentLearnMore=Learn more about adding employment information to your ORCID record
+share.addAnotherId=Add other identifier
```
Comment (DanielPalafox) [inline src/locale/properties/shared/shared.en.properties:147]: @leomendoza123 There is a typo here 
```suggestion
shared.addAnotherId=Add other identifier
```

File: src/locale/properties/shared/shared.en.properties [hunk 2]
```diff
@@ -144,3 +144,6 @@ shared.worksDescription=Add your research outputs such as publications, data set
...
+share.addId=Add an identifier
```
Comment (DanielPalafox) [inline src/locale/properties/shared/shared.en.properties:148]:
```suggestion
shared.addId=Add an identifier
```

File: src/app/types/record-funding.endpoint.ts [hunk 3]
```diff
@@ -44,7 +44,7 @@ export interface Funding extends AssertionBase {
   region: Value
   country: Value
   countryForDisplay?: string
-  fundingTypeForDisplay?: string
+  TypeForDisplay?: string
```
Comment (DanielPalafox) [inline src/app/types/record-funding.endpoint.ts:47]: there is a typo here 
```suggestion
  typeForDisplay?: string
```

**Comment by leomendoza123** [comment]
[[Design QA] Editing pre-existing identifiers on works and funding modals](https://trello.com/c/VyZ2M5ZM/7551-design-qa-editing-pre-existing-identifiers-on-works-and-funding-modals)

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

File: src/app/record/components/funding-external-identifiers-edit/funding-external-identifiers-edit.component.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<div class="top-bar">
+  <strong>
+    <ng-container i18n="@@share.identifier">Identifier</ng-container> ({{
+      index + 1
+    }})
+  </strong>
+  <a (click)="cancel()">Cancel</a>
```
Comment (DanielPalafox) [inline src/app/record/components/funding-external-identifiers-edit/funding-external-identifiers-edit.component.html:7]:
```suggestion
  <a i18n="@@shared.cancel" (click)="cancel()">Cancel</a>
```

File: src/app/record/components/funding-external-identifiers-view-only/funding-external-identifiers-view-only.component.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+  <app-panel-data-line
+    *ngIf="grant?.fundingRelationship"
+    [title]="fundingRelationshipLabel"
+    [content]="grant.fundingRelationship"
+  >
+  </app-panel-data-line>
+</div>
+<div class="controls">
+  <a (click)="edit.emit()">Edit</a>
```
Comment (DanielPalafox) [inline src/app/record/components/funding-external-identifiers-view-only/funding-external-identifiers-view-only.component.html:24]:
```suggestion
  <a i18n="@@shared.edit" (click)="edit.emit()">Edit</a>
```

File: src/app/record/components/funding-external-identifiers-view-only/funding-external-identifiers-view-only.component.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  <a (click)="delete.emit()">Delete</a>
```
Comment (DanielPalafox) [inline src/app/record/components/funding-external-identifiers-view-only/funding-external-identifiers-view-only.component.html:25]:
```suggestion
  <a i18n="@@shared.delete" (click)="delete.emit()">Delete</a>
```

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Pull request merged**